.gitignore

```diff
@@ -27,3 +27,19 @@ missing
 stamp-h1
 aac-enc
 compile
+
+CMakeCache.txt
+CMakeFiles
+CMakeScripts
+Testing
+Makefile
```

Comment:
"Makefile" is duplicated here. What creates a file/directory named "Testing"? I have a vague feeling that this contains a fair bit of more generated files than I normally see in a cmake build. (And for the case of an out-of-tree build, or building in a separate builddir, as the comments in the cmake file suggests, I guess none of these would end up generated here at all?)